xlators/protocol/server/src/server.c

```diff
@@ -627,12 +682,40 @@ server_graph_janitor_threads(void *data)
                " %s stack",
                victim->name);
         xlator_mem_cleanup(victim);
+        LOCK(&ctx->volfile_lock);
+        {
+            totchildcount = 0;
+            for (trav_p = &top->children; *trav_p; trav_p = &(*trav_p)->next) {
+                totchildcount++;
```

Comment:
We don't really need to know how many children we have. As soon as one child is detected, we can exit the loop.

Reply:
done